ui/src/components/timeline/QasTimeline.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <q-timeline
+    class="qas-timeline"
+    color="grey-6"
+    layout="comfortable"
+  >
+    <template
+      v-for="(item, index) in list"
+      :key="`timeline-${index}-${item.id}`"
```

Comment:
To preocupado com o `.id` aqui, se a gente for usar ele aqui precisa estar documentado que todo registro da timeline precisa ter um `id`...

Você lembra o motivo dele ser necessário? Acho que dava bug quando deletava um item da timeline né?

Reply:
feito conforme conversamos no discord